### PR TITLE
Minor cleanups to resource code.

### PR DESCRIPTION
**Pull request by dlorenc**
When starting work on a prototype for a new resource, I noticed a few small issues:
- typos in comments
- an unused method on cluster_resource.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md#commit-messages)

# Release Notes

### CODE REVIEW

File: pkg/apis/pipeline/v1alpha1/cluster_resource.go [hunk 1]
```diff
@@ -118,12 +118,9 @@ func (s *ClusterResource) GetURL() string {
 	return s.URL
 }
 
-// GetParams returns the resoruce params
+// GetParams returns the resource params
 func (s ClusterResource) GetParams() []Param { return []Param{} }
 
-// GetSecrets returns an array with field name and the corresponding secret to populate it
-func (s ClusterResource) GetSecrets() []SecretParam { return s.Secrets }
```
Comment (vdemeester) [inline pkg/apis/pipeline/v1alpha1/cluster_resource.go]: Isn't it there to satisfy an interface though ? :thinking:

Comment (dlorenc) [reply pkg/apis/pipeline/v1alpha1/cluster_resource.go]: Maybe at one point it was? I can't find any interfaces that require it though :(

And it looks like it builds without it, so 🤷‍♂️ ?

Comment (vdemeester) [reply pkg/apis/pipeline/v1alpha1/cluster_resource.go]: > And it looks like it builds without it, so man_shrugging ?

I accept that :100:

**Comment by bobcatfish** [comment]
_yesssss_ clean up is the best

**Pull request merged**